pgmpy/tests/test_readwrite/test_ProbModelXML.py

```diff
@@ -635,7 +637,48 @@ def setUp(self):
+                                                                                     {'Thresholds': [{'value': '0', 'belongsTo': 'Left'},
+                                                                                                     {'value': '+Infinity'}],
+                                                                                      'Potential': {'Subpotentials': [{'NumericVariables': ['C1'],
+                                                                                                                       'Potential': {'Variables': {'D1': []},
+                                                                                                                                     'type': 'Table',
+                                                                                                                                     'Values': '10  5'},
+                                                                                                                       'Coefficients': '0.25',
+                                                                                                                       'type': 'Exponential'}],
+                                                                                                    'Variables': {'C1': ['D1']},
+                                                                                                    'type': 'MixtureOfExponentials'}}],
+                                                                        'TopVariable': 'C1'},
+                                                          'States': [{'name': 'no'}]},
+                                                         {'Potential': {'Subpotentials': [{'NumericVariables': ['C0'],
+                                                                                           'Potential': {'type': 'Table',
+                                                                                                         'Values': '0.3'},
+                                                                                           'Coefficients': '1',
+                                                                                           'type': 'Exponential'},
+                                                                                          {'Potential': {'type': 'Table',
+                                                                                                         'Values': '0.7'},
+                                                                                           'type': 'Exponential'}],
+                                                                        'Variables': {'C0': []},
+                                                                        'type': 'MixtureOfExponentials'},
+                                                          'States': [{'name': 'yes'}]}],
+                                            'TopVariable': 'D0'}]}}
```

Comment:
All these lines above in `test_potential`, please make them at least somewhat pep8 compatible, if not 100% pep8.